tasks/deploy_start.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Send initial deployment notification to Slack
+  slack:
+    token: "{{ slack_webhook_token }}"
+    channel: slack_webhook_channel
+    attachments:
+      - text: "*{{ lookup('pipe', 'git config user.name') }}* has started deploying {{ site }} to *<https://{{ project.site_hosts | map(attribute='canonical') | first }}|{{ env }}>*"
```

Comment:
Does it support Bedrock in a seperate repo?
i.e: https://github.com/roots/trellis/blob/f11a7cefbc978831d1ff6b4156298e618ff2ff80/group_vars/production/wordpress_sites.yml#L12 is not the same as the Trellis repo.


Reason: We separate Bedrock and Trellis repos. See: https://github.com/roots/trellis/issues/883#issuecomment-329052189

Reply:
I'd assume so.. but that might come back to bite me! I'll do a couple tests sometime on Monday!

Reply:
I'm pretty sure this should be fine as the site variables would available no matter how you're splitting up your project